### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SkylarPlayz348**
Made what I though were things I would want to know to the usage part of the readme and formatted it a bit

### CODE REVIEW

**Review by FAXES — COMMENTED** [review]
Waiting for a review from @braxtongpoll to read over as he does more of the better usage related info from me.

However, with this said maybe updating to the extension used in the default config files might be a good change so everything is uniform.

**Review by braxtongpoll — APPROVED** [review]
Apologize for the delay! Looks good to go.

**Pull request merged**
